docs/source/configuration.rst

```diff
@@ -1010,15 +1010,29 @@ VCard is taken, and if that is not set but `muc_nickname_from_jid`_ is set to
 
 If no nickame value is found, then an error will be raised.
 
-muc_hats_from_vcard
```

Comment:
Please add something to the CHANGELOG to indicate that a backwards incompatible change is made by renaming a configuration setting.

Reply:
And done.